distribution/pom.xml

```diff
@@ -171,7 +171,6 @@
 
     <modules>
         <module>fully-loaded</module>
```

Comment:
I agree. I think those optional dependencies are confusing. Seeing e.g. groovy, mustache, expressions as optional is really confusing: if we really want them optional, why aren't they scripting plugins? I guess the answer is that we want these in the server by default (hence, included in distribution/xxx), and the optional is a hack for users that are using elasticsearch.jar in some other way. 

So one alternative in the future is for them to just be plugins that are "pre-installed" in the distribution/xxx versus optional jars that are "pre-installed"... but thats a whole nother mess to figure out (requires changing how we do integration tests for plugins)